### DISCUSSION TITLE
AMD GPU support on Windows

### DISCUSSION THREAD

**Discussion by Kleankyrie2** [oobabooga/text-generation-webui#2813] [General]
When can we expect support for AMD GPU on Windows? Is it possible to have the same support like for stable diffusion webui?

**Comment by KeulzZz** [comment]
that would be so great...

**Comment by D-Duck** [comment]
AMD just released ROCm on Windows so my guess is it will come sooner or later for these cards https://rocm.docs.amd.com/en/latest/release/windows_support.html

**Comment by catin-black** [comment]
+1 I would be more than happy if you will be able to do it. AUTOMATIC1111 works with AMD and Windows without problem right now.
